src/constants.ts

```diff
@@ -2,3 +2,5 @@ export const FORTA_BOT_REGISTRY: string = "0x61447385B019187daa48e91c55c02AF1F1f
 export const BOT_DEPLOYER_ADDRESS: string = "0x88dC3a2284FA62e0027d6D6B1fCfDd2141a143b8";
 export const NEW_AGENT_FUNCTION_SIGNATURE: string =
   "function newAgent(uint256 agentId, address owner, string metadata, uint256[] chainIds) public";
+export const TEST_AGENT_FUNCTION_SIGNATURE: string =
```

Comment:
I think changing the variable name to something like `INCORRECT_FUNCTION_SIGNATURE` could be a bit more clear as to its purpose. Just a small suggestion.